packages/cdk/lambda-python/generic-agent-core-runtime/mcp-configs/agent-builder/mcp.json

```
@@ -74,6 +74,14 @@
         "category": "Search",
         "description": "Web search and research capabilities powered by Tavily"
       }
+    },
+    "tavily-gateway": {
+      "command": "uvx",
+      "args": ["mcp-proxy-for-aws", "<AgentCore Gateway Endpoint>"],
```

```suggestion
      "args": ["mcp-proxy-for-aws", "<AgentCore Gateway resource URL>"],
```

の方がマネコンと表記が統一できていて親切かもしれません。